projects/services/storage-service-typescript/src/index.ts

```diff
@@ -1,24 +1,42 @@
-import { ErrorHandler, grpc, ServiceRunner, topology, topologyPeers } from "orbs-core-library";
+import * as path from "path";
+
+import { logger, ErrorHandler, grpc, ServiceRunner, topology, topologyPeers } from "orbs-core-library";
 
 import BlockStorageService from "./block-storage-service";
 import StateStorageService from "./state-storage-service";
 
+const { NODE_NAME, NODE_ENV, LOGZIO_API_KEY, BLOCK_STORAGE_POLL_INTERVAL, BLOCK_STORAGE_DB_PATH } = process.env;
```

Comment:
There's a lot of redundancy here.  It makes much more sense to introduce a Logger.loadConfigFromEnv() or something of that sort